NerlnetInstall.sh

```diff
@@ -4,12 +4,19 @@
 NERLNET_LIB_DIR="/usr/local/lib/nerlnet-lib"
 NERLNET_DIR=$NERLNET_LIB_DIR/NErlNet
 NERLNET_LOG_DIR="/usr/local/lib/nerlnet-lib/log"
-LOGGED_IN_USER=$(logname)
 NumJobs=4
 InstallAll=false
 REBAR3_FILE=src_erl/rebar3/rebar3
 REBAR3_SYMLINK=/usr/local/bin/rebar3
 
+if [[ "$RUNNING_IN_DOCKER" = true ]]; then
+  # Not running inside a docker container
+  LOGGED_IN_USER=$(logname)
+else
+  # Running inside a docker container
+  LOGGED_IN_USER="$(whoami)" # Probably root
```

Comment:
@dolby360 
Is the container has its own user list? 
container's root is not the OS root I guess

Reply:
It is. In a container, you always run as root.